test/orders/service/OrdersService.spec.ts

```diff
@@ -223,6 +225,71 @@ describe('OrdersService', () => {
       });
     });
 
+    it(`And valid businessId and single status
+            When retrieving orders Then return resolved promise with orders`, () => {
+      const status = OrderStatus.OPEN;
+      const businessId = '4d688534-d82e-4111-940c-322ba9aec108';
+
+      const mockedAxiosAdapter = new axiosMockAdapter(axios);
+      mockedAxiosAdapter.onGet(
+        `${Environment.STAGING}/${Version.V1}/orders`,
+        {
+          params: {
+            status,
+            businessId
+          }
+        }
+      ).reply(200, {
+        orders: multipleOrdersStatus.filter(order => order.status === OrderStatus.OPEN),
+        pagination: {
+          lastPage: true
+        }
+      });
+
+      const requestOptions: OrdersQueryRequestByBusinessId = {
+        businessId,
+        status
+      };
+      return ordersService.get(requestOptions).then((actualOrders) => {
+        expect(actualOrders).to.be.an('array');
+        expect(actualOrders.length).to.equal(1);
+        expect(actualOrders[0].businessId).to.equal(businessId);
+      });
+    });
+
+    it(`And valid businessId and multiple statuses
+            When retrieving orders Then return resolved promise with orders`, () => {
+      const status = [OrderStatus.CANCELED, OrderStatus.OPEN];
+      const businessId = '4d688534-d82e-4111-940c-322ba9aec108';
+
+      const mockedAxiosAdapter = new axiosMockAdapter(axios);
+      mockedAxiosAdapter.onGet(
+        `${Environment.STAGING}/${Version.V1}/orders`,
+        {
+          params: {
+            status,
+            businessId
+          }
+        }
+      ).reply(200, {
+        orders: multipleOrdersStatus
+          .filter(order => order.status === OrderStatus.OPEN || order.status === OrderStatus.CLOSED),
```

Comment:
Use the status variable you are using in the mock above with
 order => status.includes(order.status)

This will make it so you filters match in the request and the response.